src/implementations/vanilla/components/menu/menu.js

```diff
@@ -893,6 +901,13 @@ export class Menu {
     this.inner.setAttribute('aria-hidden', 'false');
     this.isOpen = true;
 
+    // Update label
+    if (this.toggleLabel) {
```

Comment:
I'm confused, this "label" could have already a value set, right? If we do this and these attributes are not set, or they are just empty, we would wipe this value, no? I think we should also check for the non emptyness of the attribute before using it to override this label.